bears/js/PrettierLintBear.py

```diff
@@ -0,0 +1,34 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='prettier',
+        use_stdout=True,
+        use_stderr=True)
+class PrettierLintBear:
+    """
+    Formats JavaScript,JSX,Flow,TypeScript,CSS,Less,SCSS,JSON,GraphQL,Markdown
+    files according to opinionated code format
+    using ``prettier``.
+
+    See <https://github.com/prettier/prettier> for more information.
+    """
+    LANGUAGES = {'JavaScript'}
+    REQUIREMENTS = {NpmRequirement('prettier', '1.9.2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Consistency', 'Correctness', 'Whitespace',
+                  'Parentheses', 'Strings', 'Empty lines',
+                  'Multi-line objects'}
+    SEE_MORE = 'https://prettier.io/'
+
+    def process_output(self, output, filename, file):
+        stdout, stderr = output
+        regex = r'L(?P<line>\d+)C(?P<column>\d+): (?P<message>.*)'
```

Comment:
I would recommend to move this variable to class scope and compile the expression, to save performance (CPython internally caches regexes, but I don't know how many)